src/server/services/wms/qgswmsparameters.cpp

```diff
@@ -470,6 +505,17 @@ namespace QgsWms
     : QgsWmsParameters()
   {
     load( parameters.urlQuery() );
+
+    const QString sld = mWmsParameters[ QgsWmsParameter::SLD ].toString();
+    if ( !sld.isEmpty() )
+    {
+      QByteArray ba = mWmsParameters[ QgsWmsParameter::SLD ].loadUrl();
+      if ( !ba.isEmpty() )
+      {
+        const QString sldBody = QString::fromStdString( ba.toStdString() );
```

Comment:
@rldhont Do you think that the string conversion could be hidden in the `loadUrl()` method?